crates/fuzzing/wasm-spec-interpreter/ocaml/Makefile

```diff
@@ -0,0 +1,31 @@
+# Build a library allowing FFI access to the Wasm spec interpreter.
```

Comment:
Yeah, originally I had this all inlined in the `build.rs` but I moved away from it because it made it hard to figure out what went wrong when something invariably went wrong. This way, it's relatively easy to run `make` in the `ocaml` directory to troubleshoot.

Reply:
@alexcrichton made a point today that the spec interpreter itself is built with `make` and pulling that all in to a `build.rs` is too much work so we might as well keep all of it in a `Makefile`.